rfcs/text/0000-rfc-process-updates.md

```diff
@@ -0,0 +1,134 @@
+### DCP PR:
+
+PLEASE DO NOT REVIEW! THIS RFC IS NOT COMPLETE AND IS STILL ACTIVELY BEING WORKED ON.
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC Name
+
+*Replace "RFC Name" above with the name of your RFC. Keep it short and descriptive.*
+
+## Summary
+
+The Data Coordination Platform (DCP) established the Request For Comments (RFC) process on 
+[September 17th 2018](https://github.com/HumanCellAtlas/dcp-community/pull/26) in order to address the gap of having a
+coordinated, standardized, and repeatable process by which decisions could be made. Since this time, the DCP has evolved
+and likewise the RFC process is in need of some fine tuning. This RFC proposes some changes in order to reflect the new
+DCP community's needs.  
+
+## Author(s)
+
+*Recommended format for Authors:*
+
+ [Arathi Mani](mailto:arathi.mani@chanzuckerberg.com)
+ 
+ [Dave Rogers](mailto:dave@clevercanary.com)
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+
+Over the last year since the original RFC process was accepted, the DCP project has grown both in size and complexity.
+As of this writing, 11 RFCs have been accepted in addition to many charters. Not so unexpectedly, there have been pain
+points along the way of which we try to address a few via edits to the RFC process defined below.
+
+
+### User Stories
+
+*Share the [User Stories](https://www.mountaingoatsoftware.com/agile/user-stories) motivating this RFC.*
+
+## Detailed Design
+
+
+### 1. Add additional RFC states
+
+Currently the states of an RFC is one of:
+ 
+ 1. "rfc-community-review",
+ 1. "rfc-oversight-review", 
+ 1. "rfc-approved", or
+ 1. "rfc-declined." 
+
+In practice, there have been cases where an RFC is no longer implemented (i.e. 
+"deprecated") or an RFC has been withdrawn due to either incompleteness or having been published too early. 
+
+Additionally, with the encouraged behavior of submitting draft pull requests prior to RFC readiness for review, there is no way to clearly designate the fact that the RFC should not be reviewed at all until the PR is no longer a draft.
+
+
+We propose adding the following new tags: 
+ 
+
+**pre-rfc-draft** : This indicates that the authors are still actively working on the RFC, that the RFC is in a draft state, and readers should **not** comment on the RFC yet. Additionally this state serves to indicate more clearly the backlog of RFCs in development to avoid duplication of effort of overlapping RFCs and to help elicit offers of co-authorship.
```

Comment:
Makes sense to me. I changed to `rfc-draft`. The one case for `pre-rfc-draft` is to use the term that is used colloquially (i.e. "pre-rfc") so that it's even more clear.

Reply:
Closing this thread as I believe it has been addressed, but feel free to re-open if not!